ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -261,7 +267,7 @@ def publish_to_result_queue(self, result_queue: str, result_message: OcrdResultM
             message=encoded_result_message
         )
 
-    def create_queue(self, connection_attempts=1, retry_delay=1):
+    def create_queue(self, connection_attempts: int = 1, retry_delay: int = 1):
```

Comment:
The default for connection attempts should be `config.OCRD_NETWORK_WORKER_QUEUE_CONNECT_ATTEMPTS`. The retry delay should also be a bit higher - maybe 3 seconds?

Reply:
I think it is better not to use the ENV as a function-parameter default value. Why do you think this is better?

Reply:
Makes sense, I committed accordingly.